Source/Configs/Planetary.debug.xcconfig

```diff
@@ -55,7 +55,7 @@ PLANETARY = @l1sGqWeCZRA99gN+t9sI6+UOzGcHq3KhLQUYEwb4DCo=.ed25519
 PLANETARY_SYSTEM = ${PLANETARY}
 
 ALIAS_SERVER_ONE = Planetary Alias::net:planetary.name:8008~shs:42ACINb4+Qjd1kPYYpJ67V16CTHIWp93cjd1gDX7FKY=::icon-planetary-3::planetaryRoomToken
-PLANETARY_COMMUNITY_ALIAS_SERVERS = ${ALIAS_SERVER_ONE}
+PLANETARY_COMMUNITY_ALIAS_SERVERS = ${ALIAS_SERVER_ONE}||
```

Comment:
Is this correct?

Reply:
yes, it is. we have the same weird || in the release config, which is the definitive one. It is a way of telling the parsing algorithm that it is an array of one element, not just a string